Trellix/trellix-epo/ingest/parser.yml

```
@@ -0,0 +1,58 @@
+name: trellix-epo
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: output
+
+  - name: parsed_date
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_event.output.timestamp}}"
+        output_field: datetime
+
+  - name: set_common_fields
+  - name: set_additional_trellix_fields
+
+stages:
+  set_additional_trellix_fields:
+    actions:
+      - set:
+          trellix.event.id: "{{json_event.output.autoguid}}"
+          trellix.event.detect_date: "{{json_event.output.detectedutc}}"
+          trellix.event.receive_date: "{{json_event.output.receivedutc}}"
+          trellix.agent_id: "{{json_event.output.agentguid}}"
```

I think that `agent.id` may be suitable for this information
```suggestion
          agent.id: "{{json_event.output.agentguid}}"
```